### PR TITLE
Outline Button for review

### PR DESCRIPTION
**Pull request by Usiel-Gandalf**
Name: Outline Button - UI Design System
Link: https://devaptivist.atlassian.net/browse/LP-68?atlOrigin=eyJpIjoiMzZkZWMxZjBmYzFjNDA4Mzg3MTYwNjdmN2JjMzE4OGEiLCJwIjoiaiJ9
Description: There is a outlined button without name specific, and in this button we can set a Icon depends of the use.
Outline Button preview

### CODE REVIEW

**Review by IngKevel — APPROVED** [review]
Nice work

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.unit.dp
+
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt:18]: Can you add the description of the function please? With the following criteria:

- description of the function
- description of each parameter
- author

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+@Composable
+fun LpOutlinedButton(
+    modifierButton: Modifier = Modifier,
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt]: like the modifier is a nomenclatura for the properties of component in compose, is called just modifier.

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    textButtonDescription: String,
+    icon: ImageVector? = null,
+    onClick: () -> Unit,
+) {
+    OutlinedButton(
+        onClick = onClick,
+        modifier = modifierButton
+            .padding(10.dp)
+            .fillMaxWidth(),
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt]: this `fillMaxWidth()` property could be sent by the function that is implementing this `LpOutlinedButton` composable

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        colors = ButtonDefaults.buttonColors(
+            containerColor = MaterialTheme.colorScheme.onSurface,
+            contentColor = MaterialTheme.colorScheme.secondary
+        )
+    ) {
+        Box(modifier = Modifier.fillMaxWidth(), contentAlignment = Alignment.CenterStart) {
+            Row {
+                icon?.let {
+                    Icon(
+                        icon,
```
Comment (dmroldanch) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Button.kt:50]: Example comment

**Review by melissaquezada94 — APPROVED** [review]

**Pull request merged**